independent-publisher-connectors/DPIRD Weather/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3611 @@
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Air temp low minutes current month",
+                    "description": "The number of minutes that the air temperature was less than or equal to 2 degrees celsius in the current month."
+                  },
+                  "startTime": {
+                    "type": "string",
+                    "title": "Air temp low start date current month",
+                    "description": "The date/time of the first occurrence of the air temperature being less than or equal to 2 degrees celsius in the current month."
+                  },
+                  "days": {
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Air temp low days current month",
+                    "description": "The number of days the air temperature was less than or equal to 2 degrees celsius in the current month."
+                  }
+                },
+                "description": "The total number of minutes that the air temperature was less than or equal to 2 degrees celsius in the current month and the number of days it has occurred on in this period."
+              },
+              "yearToDate": {
+                "type": "object",
+                "properties": {
+                  "minutes": {
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Air temp low minutes current year",
+                    "description": "The number of minutes that the air temperature was less than or equal to 2 degrees celsius in the current year."
+                  },
+                  "startTime": {
+                    "type": "string",
+                    "title": "Air temp low start time current year",
+                    "description": "The date/time of the first occurrence of the air temperature being less than or equal to 2 degrees celsius in the current year."
+                  },
+                  "days": {
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Air temp low days current year",
+                    "description": "The number of days the air temperature was less than or equal to 2 degrees celsius in the current year."
+                  }
+                },
+                "description": "The total number of minutes that the air temperature was less than or equal to 2 degrees celsius in the current year and the number of days it has occurred on in this period."
+              }
+            },
+            "description": "The number of minutes that the air temperature was less than or equal to 2 degrees celsius. The start time is available for the current day and the number of days is available for longer periods. Only calculated for stations that have an air temperature probe at a height that is less than or equal to 3 metres [m] above ground."

```

Comment:
All descriptions, titles, summaries, etc, should be free from grammar and spelling errors. Please review them. We recommend using spell checking tools.

Reply:
I ran this through tools as described. Interested in what is being reported as exception? I also changed spelling to US English in an attempt to reduce false positives for you

Reply:
Metres is flagged as a misspelling. It should be "meters".

Reply:
This is correct spelling for all other jurisdictions other than US and the API is an Australian government agency API. @natalie-pienkowska  what do you think? Implication is all API metadata is US English, meaning I cannot cut/paste from the original API which describes everything in UK English